build/entrypoint.sh

```diff
@@ -3,13 +3,13 @@
 # If we are using podman or docker compose use the repo config.toml
 if [[ -f /podman_compose/kai/config.toml ]]; then
 	cp /podman_compose/kai/config.toml /kai/kai/config.toml
-	sed -i "s/^host =.*/host = \"${POSTGRES_HOST}\"/g" /kai/kai/config.toml
```

Comment:
note to self i need to update these on https://github.com/konveyor/operator/pull/374